lib/workers/repository/onboarding/branch/index.ts

```diff
@@ -72,6 +72,7 @@ export async function checkOnboardingBranch(
     }
   }
   if (!GlobalConfig.get('dryRun')) {
+    logger.debug('Checkout onboarding branch.');
```

Comment:
this is not a typo 😉

Reply:
Couldnt think of a good title that included both :(